diff --git a/node_modules/@squareetlabs/capacitor-subscriptions/Package.swift b/node_modules/@squareetlabs/capacitor-subscriptions/Package.swift
index ae48933..8c5184f 100644
--- a/node_modules/@squareetlabs/capacitor-subscriptions/Package.swift
+++ b/node_modules/@squareetlabs/capacitor-subscriptions/Package.swift
@@ -10,7 +10,7 @@ let package = Package(
             targets: ["SubscriptionsPlugin"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "7.0.0")
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", "7.0.0"..<"9.0.0")
     ],
     targets: [
         .target(
